extensions/interactions/rules.json

```diff
@@ -0,0 +1,186 @@
+{
+  "CodeRepl": {
```

Comment:
Maybe the [static-module](https://www.npmjs.com/package/static-module) npm package is something to look into here. It allows you to define modules of functions that are evaluated statically at build time. 

So for instance, you could define a `config` module, with a `getOption` function, and use it like this:

    var config = require("config");
    $(document.body).html("<h1>" + config.getOption("applicationTitle")+"</h1>");

And so after the build script has run, this gets converted into:

    var config = require("config");
    $(document.body).html("<h1>" +"Oppia"+"</h1>");

I have a browserify transforming function that I use for this in one of my previous projects, so maybe it's useful if I port that over. It means that only the necessary values or copied to the frontend, though it might increase the file size if config values are large, or repeated a lot.